lib/ex_format.ex

```diff
@@ -416,7 +416,15 @@ defmodule ExFormat do
 
   # Tuple containers
   def to_string({:{}, _, args} = ast, fun, state) do
-    tuple = "{" <> tuple_to_string(args, fun, state) <> "}"
+    inside_list? = Map.get(state, :inside_list?)
```

Comment:
Add `inside_list?: false` when initializing the map, that's def the way to go yes.